Assets/Scripts/Combat/StatusEffect.cs

```diff
@@ -18,6 +19,15 @@ public enum EffectType
     int powerLevel;
     public bool expired = false;
 
+    static public bool HasEffectType(CreatureMechanics check, EffectType effectType)
```

Comment:
In this function better pass it directly the statusEffects list.
It doesn't need all the CreatureMechanics object.

Reply:
Fair enough, done.